java/vector/src/main/java/org/apache/arrow/vector/types/pojo/Field.java

```diff
@@ -84,12 +84,18 @@ public Field(String name, boolean nullable, ArrowType type, List<Field> children
 
   // deprecated, use FieldType or static constructor instead
   @Deprecated
-  public Field(String name, boolean nullable, ArrowType type, DictionaryEncoding dictionary, List<Field> children) {
+  public Field(
+      String name,
+      boolean nullable,
+      ArrowType type,
+      DictionaryEncoding dictionary,
+      List<Field> children) {
     this(name, new FieldType(nullable, type, dictionary, null), children);
```

Comment:
I'm not sure if this specific style of indentation is enforced by our checkstyle, but wanted to bring it up in case others had different preferences.

Reply:
I prefer this format. FYI I believe intellij does a different one where arguments hang off the opening parenthesis but I much prefer this pattern. +1 on enforcing it.

Reply:
Sounds good, I'll try to see if I can figure out how to enforce it. It's hard to tell right now because there are so many other warnings.